src/xo-app/backup/new/index.js

```diff
@@ -261,10 +261,19 @@ export default class New extends Component {
 
     if (values.length === 1) {
       // Older versions of XenOrchestra uses only values[0].
-      values = values[0].values
-
-      backupInput.value = values[0]
-      vmsInput.value = { vms: values }
+      const array = values[0].values
+      const config = array[0]
+      const reportWhen = config._reportWhen
+
+      backupInput.value = {
+        ...config,
+        _reportWhen:
+          // Fix old reportWhen values...
+          (reportWhen === 'fail' && 'failure') ||
+          (reportWhen === 'alway' && 'always') ||
```

Comment:
My bad, I read this code incorrectly, I apologize for all the fuss, good work @Wescoeur :smiley: